Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeStats.h

```diff
@@ -0,0 +1,55 @@
+//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+//See LICENSE in the project root for license information.
+
+#pragma once
+
+using namespace Platform::Collections;
+
+BEGIN_NAMESPACE_GAZE_INPUT
+
+private ref class GazeStats sealed
```

Comment:
I see this was changed to private, but I don't see the class referenced anywhere. Can we just delete it?

Reply:
We will be very shortly (immediately after this release) adding stuff that needs this. Is it okay to leave it in till then?

Reply:
Sounds good. Just checking :)